lib/server.js

```diff
@@ -14,9 +14,13 @@ process.on('unhandledRejection', (reason, p) => {
  * @param  {Hapi.Request}    request Hapi Request object
  * @param  {Hapi.h}     h   Hapi Response Toolkit
  */
-function prettyPrintErrors(request, h) {
+function prettyPrintErrors(request, h, release) {
     const { response } = request;
 
+    if (release && release.mode === 'canary' && !(request.state && request.state[release.cookieName])) {
```

Comment:
I think we can get the cookie value of `release` from `request.state.data`.

Reply:
And get `release` value from `config.release` rather than a passing parameter. 

So this code does not have to change.

`server.ext('onPreResponse', prettyPrintErrors);`